centos.org/jobs/foreman-pipelines.yml

```diff
@@ -16,6 +16,7 @@
       - '2.4'
       - '2.5'
       - '3.0'
+      - '3.1'      
```

Comment:
This will include Ubuntu 18.04. You should modify the nightly config below. I'm not sure about the naming

Reply:
I already removed the Ubuntu 18.04 `theforeman.org/pipelines/vars/foreman/3.1.groovy` , isn't the VAR supposed to bring only the SO that is declared inside the 3.1.groovy file?

Reply:
Technically you can add it to `foreman-nightly` and rename it to `foreman-31-and-newer`. The name isn't really used anyway.

Reply:
I already did on  d2fe358841ae234a43a670b080cc2e069f137f61 , I can change it before the rebase/squash